### ISSUE TITLE
Automation Bypass crashes on automation overlay

### ISSUE THREAD

**Issue by chip-nerd** [Berkays/OniMods#5]
When switching to automation overlay (Shift+F2), the game (Spaced Out edition) crashes:

```
MissingMethodException: System.Collections.IEnumerator KCompactedVector`1.GetEnumerator()
(wrapper dynamic-method) OverlayModes+Logic.OverlayModes+Logic.UpdateUI_Patch2(OverlayModes/Logic)
OverlayModes+Logic.Update () (at <990b4cf9a7d84305ad7eaef1059b672b>:0)
OverlayScreen.LateUpdate () (at <990b4cf9a7d84305ad7eaef1059b672b>:0)
Build: U47-561558-SD
```

The issue appeared after the game update to June 2023 Quality of Life.

**Comment by Berkays** [comment]
Thank you for the feedback. Released 1.0.3 which hopefully fixed the issue.

**Comment by chip-nerd** [comment]
It's working now. Thank you very much for the lightning speed fix! :-)
